wbooks/src/app/components/HeaderBackground/index.tsx

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+import { Image } from 'react-native';
+import bcImage from '@assets/general/bc_nav_bar.png';
+
+import styles from './styles';
+
+function HeaderBackground() {
+  return <Image style={styles.image} source={bcImage} />;
```

Comment:
Same here, don't forget about `resizeMode`. Maybe `resizeMode="stretch"`?

Reply:
I tried with all combinations using resizeMode, but the result was the same

Reply:
Done!